water_comes/hollowings.py

```diff
@@ -6,26 +6,29 @@
     replaceColor,
 )
 import numpy as np
+import base64
+from io import BytesIO
 
+def addressToImages(address=None, x=None, y=None):
+    if address is None and (x is None or y in None):
```

Comment:
Lav evt. en raise ValueError("Must either specify address or coordinates") istedet for en return. Ellers kan det godt blive en svær bug af finde senere